Resources/AlarmTestsResources/ahuFailedToTurnOffAlarmResources.robot

```diff
@@ -0,0 +1,86 @@
+Library    JSONLibrary
+Library    Collections
+Library     String
+Library    SeleniumLibrary
+Library    ${EXECDIR}/ExternalKeywords/common.py
+Variables    ${EXECDIR}/Configurations/${environment}.py
+Resource    ${EXECDIR}/Resources/apiresources.robot
+Resource    ${EXECDIR}/Resources/connection.robot
+Resource    ${EXECDIR}/Resources/common.robot
+Resource    ${EXECDIR}/Inputs/testInputs.robot
+Resource    ${EXECDIR}/Inputs/GraphQL/gqlQueries.robot
+
+
+*** Keywords ***
+ahuFailedToTurnOffAlarmTestPreconditionSetup
+    [Documentation]    Stop all VEM processes and wait for 2 minutes
+    ...                Also write test entry temperature for the parallel staleStatePrevention program
+    log to console    !-----Reading the inputs from the excel and storing in dictionary------!
+    testInputs.readingInputsFromExcel  alarmTest  A  B
+    common.setFlagValue    ${test_entry_flag}
+    log to console    !-----PreCondition for the Alarm-1 test is been executed------!
+    connection.establishConnectionAndStopAllProcessesExcept
+    common.waitForMinutes    2
+    apiresources.writeTestEntryTemperatureToSensorsAfterVXServerStarted
+
+setAllowNumExceedencesGuardCATGuardBandRangeOnPwrLvl
+    apiresources.changeCxConfigsTabModuleFieldValues  DASHM  AllowNumExceedencesGuard  ${test_input}[allow_num_exceedences_guard]
+    apiresources.changeCxConfigsTabModuleFieldValues  DASHM  CATGuardBandRange  ${test_input}[cat_guard_band_range]
+    ${ahuoid}=    apiresources.getOidOfComponentUsingComponentName   ${test_input}[ahu_change_onpwrlvl]
+    log to console     ${ahuoid}
+    apiresources.setComponentPropertyValue   ${ahuoid}     OnPwrLvl    float      ${test_input}[ahu_property_onpwrlvl]
+
+verifyNoAlarmAHUFailedToTurnOffRaised
+    ${query}=    gqlQueries.getAlarmMsgAHUFailToTurOff
+    ${json_dictionary}=  gqlFetchJsonResponseFromQuery     ${query}
+    log to console  verify no alarm is raised i.e no message ${json_dictionary}
+    should be equal as strings  ${json_dictionary["data"]}  None
+    log to console  ================No Alarm-AHUFailedToTurnOff- found===========================
+
+
+ahuFailedToTurnOffAlarmsRaised
+    @{alarmraisedforallAhu}=    apiresources.queryToFetchAlarmMsgAHUFailToTurOff
+    #fetch the count of messages for Ahu
+    ${total}=    get length   ${alarmraisedforallAhu}
+    log to console     ================Comaparing the actuall Alarm message with expected=================
+    FOR    ${i}    IN RANGE   ${total}
+        should contain   ${alarmraisedforallAhu[${i}]}    in Control Group NoBindings is not turning OFF when commanded.
+        log to console  actual message ${alarmraisedforallAhu[${i}]} contains the expected AHU NB-AHU in Control Group NoBindings is not turning OFF when commanded.
+    END
+
+verifyAHUMismatchForAHUFailToTurOffAlarm
+    #creating the expected list of messages for the AHU
+    @{alarm_mismatch_list_expected}=    Create List      10  11  12  13  14  15  16  17
```

Comment:
If the expected list is required, we can shift this to test_Inputs excel.
If this check is applicable to all AHUs, then this variable can be assigned by fetching the AHU names list in that specific group using keyword - getAHUNamesListOfGroup

Reply:
Done - used getAHUNamesListOfGroup